pkg/controller/collection/collection_controller.go

```diff
@@ -613,6 +614,8 @@ func activate(collectionResource *kabanerov1alpha1.Collection, collection *Colle
 		// Retrieve manifests as unstructured
 		manifests, err := GetManifests(pipeline.Url, pipeline.Sha256, renderingContext, log)
 		if err != nil {
+			log.Error(err, errorMessage, "resource", pipeline.Url)
+			collectionResource.Status.StatusMessage = errorMessage + ": " + err.Error()
```

Comment:
I think we should use a different error message here - if there was no previous version, it's not an upgrade, just an activate.  Maybe leave the old errorMessage where it was, and put a new one here inside this scope?

Reply:
ok, updated as such